packages/css/src/components/error-message/error-message.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset {
+  box-sizing: border-box;
+  margin-block: 0;
+}
```

Comment:
For my understanding: we add a new reset for every component. Despite there being variations, there's also a lot of duplication. Why are we not handling some of these common resets globally? With different types of resets (i.e. block and inline resets, etc...)?

Reply:
Good point, we should look at extracting (parts of) these resets. It's a bit of a chore though, there are a lot of variations and it's easy to turn this into something harder to follow than it is now. [I've added a ticket for this](https://gemeente-amsterdam.atlassian.net/browse/DES-791)